### ISSUE TITLE
Incorrect vulnerability details for jackson-mapper-asl

### ISSUE THREAD

**Issue by OrangeDog** [OSSIndex/vulns#47]
**Vulnerability URLs**
```
https://ossindex.sonatype.org/vuln/3f596fc0-9615-4b93-b30a-d4e0532e667f
https://ossindex.sonatype.org/vuln/1205a1ec-0837-406f-b081-623b9fb02992
https://ossindex.sonatype.org/vuln/b85a00e3-7d9b-49cf-9b19-b73f8ee60275
https://ossindex.sonatype.org/vuln/ab9013f0-09a2-4f01-bce5-751dc7437494
https://ossindex.sonatype.org/vuln/4f7e98ad-2212-45d3-ac21-089b3b082e6c
https://ossindex.sonatype.org/vuln/af22d349-929c-41ae-ba6c-2f7b507026f0
https://ossindex.sonatype.org/vuln/292c11e9-cf66-4d76-aaf7-b63a091f8891
https://ossindex.sonatype.org/vuln/fc1e8802-77e5-458f-b987-eb778c6ac2fc
https://ossindex.sonatype.org/vuln/3242fdc1-bfe9-46a6-af0c-0b8f57f56eb7
https://ossindex.sonatype.org/vuln/6ce886d0-2dfd-4cef-b9a4-2fb400baf5ef
```

**Description**
These say they apply to jackson-mapper-asl. I'm sure not all of them do, and perhaps even none of them.
For example CVE-2017-7525 has the range `(,2.6.7.1)` for it, but the latest `jackson-mapper-asl` is only 1.9.13.

See also https://github.com/jeremylong/DependencyCheck/issues/2295

**Comment by msymons** [comment]
@OrangeDog, I investigated this back in 2017 and reached out to the authors of jackson for some clarification.

Basically:

1. Jackson 1.x called databinding part "mapper" (there's both mapper-asl and mapper-lgpl, the only difference being which LICENSE file/declaration was included).
2. Polymorphic deserialization was added in 1.5.
3. No matter what the CVEs state, it is safest to assume that any vulnerability that is claimed to affect versions of jackson-databind "before 2.x" will also affect `org.codehaus.jackson:jackson-mapper-asl`
and `org.codehaus.jackson:jackson-mapper-lgpl` from v1.5.0 to v1.9.13
